app/src/main/java/com/example/demoquizwithandroid/QuizViewModel.kt

```diff
@@ -8,38 +8,38 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import io.appwrite.Client
 import io.appwrite.exceptions.AppwriteException
-import io.appwrite.services.Database
+import io.appwrite.services.Databases
 import kotlinx.coroutines.launch
-import org.json.JSONArray
-import org.json.JSONObject
 
 class QuizViewModel : ViewModel() {
-    private val collectionId = "606d5bd9626ae"
+    private val url = App.appResources!!.getString(R.string.URL)
+    private val projectId = App.appResources!!.getString(R.string.PROJECT_ID)
+    private val databaseId = App.appResources!!.getString(R.string.DATABASE_ID)
+    private val collectionId = App.appResources!!.getString(R.string.COLLECTION_ID)
     lateinit var client : Client
     fun create(context: Context) {
         client = Client(context)
-            .setEndpoint("https://demo.appwrite.io/v1")
-            .setProject("606d5bc9de604")
+            .setEndpoint(url)
+            .setProject(projectId)
+            .setSelfSigned(true)
         getQuestions()
     }
     private val db by lazy {
-        Database(client)
+        Databases(client)
     }
 
-    private val _questions = MutableLiveData<JSONArray>().apply { value = null }
+    private val _questions = MutableLiveData<List<QuestionModel>>().apply { value = null }
 
-
-    val questions: LiveData<JSONArray> = _questions;
+    val questions: LiveData<List<QuestionModel>> = _questions
     val selectedQuestion = MutableLiveData<Int>().apply { value = 0}
     val correct = MutableLiveData<Int>().apply { value = 0 }
 
     private fun getQuestions() {
         viewModelScope.launch {
             try {
-                var response = db.listDocuments(collectionId)
-                val json = response.body?.string() ?: ""
-                var que = JSONObject(json)
-                _questions.postValue( que["documents"] as JSONArray)
+                val response = db.listDocuments(databaseId = databaseId, collectionId = collectionId)
+                val questions : List<QuestionModel> = response.documents.map { QuestionModel(question = it.data["question"] as String, options = it.data["options"] as List<String>, answer = it.data["answer"] as String) }
```

Comment:
Let's break this into multiple lines so it's easier to read

```suggestion
                val questions: List<QuestionModel> = response.documents.map { 
                    QuestionModel(
                        question = it.data["question"] as String, 
                        options = it.data["options"] as List<String>, 
                        answer = it.data["answer"] as String
                    )
                }
```